Python Basics/python_basics_home_task.py

```diff
@@ -0,0 +1,38 @@
+# Home task for Module 1 - Python Basics
+# Create list of 100 random numbers from 0 to 1000
+import random  # Import of random module for using of random function
+
+list_of_random = []  # Define the empty list
```

Comment:
```suggestion
random_numbers = []  # Define the empty list
```
It's not clear from your variable name random what? I think random_numbers for instance will show what it does contain.